flocker/restapi/_infrastructure.py

```diff
@@ -110,6 +110,35 @@ def failure(reason):
     return logger
 
 
+def _remote_logging(original):
+    """
+    Decorate a method which implements an API endpoint to do Eliot-based log
+    tracing; that is, the ability to continue a remote task.
+
+    The remote task's context will be extracted from a C{X-Eliot} HTTP header.
+
+    :param original: Function to wrap.
+
+    :return: Wrapped function.
+    """
+    @wraps(original)
+    def logger(self, request, **routeArguments):
+        serialized_remote_task = request.requestHeaders.getRawHeaders(
+            "X-Eliot", [None])[0]
```

Comment:
Any reason not to use the header used in the example from the documentation (`X-Eliot-Task-Id`)?